### PR TITLE
Update locale es_ES (Spanish)

### PR DESCRIPTION
**Pull request by ginodeis**
Hi,

First of all, thanks and congratulations for this great project.

I have found some translation problems using the package with es_ES locale and continuing with the PR [#1471](https://github.com/ant-design/ant-design/issues/1471) I have tried to correct them.

I have updated [rc-pagination](https://github.com/react-component/pagination/pull/73) component too, to support es_ES locale.

Thanks @benjycui for your help.

### CODE REVIEW

File: components/locale-provider/es_ES.tsx [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
-    justOkText: 'OK',
+      okText: 'OK',
+      cancelText: 'Cancelar',
+      justOkText: 'OK',
   },
   Popconfirm: {
-    okText: 'OK',
-    cancelText: 'Cancelar',
+      okText: 'OK',
+      cancelText: 'Cancelar',
```
Comment (afc163) [inline components/locale-provider/es_ES.tsx]: You shouldn't modify the indent.

Comment (ginodeis) [inline components/locale-provider/es_ES.tsx]: Sorry, I'm stupid.

**Comment by enzolutions** [comment]
Hi @afc163 && @ginodeis 

I am interested in this PR to be part of next release if there is a guide to follow to resolve the CI issues I could help to address.

Kind regards,

**Comment by benjycui** [comment]
👍

**Comment by enzolutions** [comment]
Hi @benjycui 

Thanks, I have a question regards released the 2.10 will be release after 2.9.3?

Kind regards,

**Comment by benjycui** [comment]
We will release antd@2.10 in a week

**Comment by enzolutions** [comment]
@ginodeis 

Thanks for your contribution, with the new version 2.10.0 now the calendar is working in Spanish.

But, I have a concern about the basic label like validation "% is required" is not in Spanish despite it's inside a Local Provider.

Any suggestions?

Buen trabajo.

**Comment by benjycui** [comment]
@enzolutions validation's locales are not under the control of `LocaleProvider`, more: https://github.com/yiminghe/async-validator/

**Comment by enzolutions** [comment]
@benjycui 

Thank for your response.

but "% is required" and other are inside antd code, should be possible to contribute a translation of that standard message?

Any ideas?

**Comment by benjycui** [comment]
@enzolutions https://github.com/ant-design/ant-design/issues/6082

**Pull request merged**